### ISSUE TITLE
Minifiying `aria-hidden="true"` to `aria-hidden` is not recommended

### ISSUE THREAD

**Issue by pbru87** [trullock/NUglify#230]
NUglify minifies `aria-hidden="true"` to `aria-hidden`.

According to [Font Awesome](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility), this should not be done:

> **Using property shorthand**
You might be tempted to use <i aria-hidden></i> and leave the ="true" part out. Don't, it's important for screen readers to have a value that follows the spec.

If you run a website with just  `aria-hidden` against [Google Lighthouse](https://developers.google.com/web/tools/lighthouse), it also recommends to use the non-minified version `aria-hidden="true"` instead.

**Comment by trullock** [comment]
Are you aware of any screen readers which do require the `="true"` for them to work properly? NUglify takes a very aggressive (but should be functional) approach to minification, much of the output is debatably not spec-compliant, but is correctly parsed by browsers.

Happy to "fix" this if its a real problem.

Can you repro an issue?

**Comment by pbru87** [comment]
Hi Andrew, I have just set up a test web page, which can be found at [https://www.patrick-brunck.de/temp/accessibility-test.html](https://www.patrick-brunck.de/temp/accessibility-test.html)

The page contains test case 1 (`aria-hidden="true"`) and test case 2 (`aria-hidden`).

Please open the web page via **Mozilla Firefox**, [switch to the **reader view** and use its **screen reader** feature](https://support.mozilla.org/en-US/kb/firefox-reader-view-clutter-free-web-pages).

Test case 1 (`aria-hidden="true"`) will not be read, but test case 2 (`aria-hidden`).

I hope this helps!

Best regards,
Patrick

**Comment by trullock** [comment]
Ok great, are there other aria attributes which require the same treatment?

**Comment by pbru87** [comment]
None that I know about. I just stumpled on this issue, because of the feedback I got from **Google Lighthouse**.

**Comment by trullock** [comment]
Fixed in 1.13.2
